client/a8c-for-agencies/sections/marketplace/products-overview/product-card/index.tsx

```diff
@@ -76,6 +76,16 @@ export default function ProductCard( props: Props ) {
 		}
 	}, [] );
 
+	const truncateDescription = ( description: any ) => {
+		if ( description.length <= 130 ) {
+			return description;
+		}
+
+		const lastSpace = description.slice( 0, 128 ).lastIndexOf( ' ' );
+
+		return description.slice( 0, lastSpace > 0 ? lastSpace : 129 ) + '…';
+	};
+
```

Comment:
This is a nit pick. But I think you can simplify this by just:
```suggestion
	const truncateDescription = ( description: any ) => {
		const lastSpace = description.slice( 0, 128 ).lastIndexOf( ' ' );
		return description.slice( 0, lastSpace <= 129 ? lastSpace : 128 ) + '…';
	};

```